mmv1/templates/terraform/examples/region_url_map_default_route_action.tf.erb

```diff
@@ -0,0 +1,101 @@
+resource "google_compute_region_url_map" "<%= ctx[:primary_resource_id] %>" {
+  region = "us-central1"
+
+  name        = "<%= ctx[:vars]['region_url_map_name'] %>"
+  description = "a description"
+
+  default_route_action {
```

Comment:
Can you write a test that updates several of the fields in here (particularly the lists like retry_policy & weighted_backend_services) to add items, remove items and/or remove the list entirely?

Updates on these sorts of fields can be complicated and produce perma-diffs for users depending on how the API is implemented, and we lose a lot of possible fixes once we release the resource with a certain implementation

Reply:
Will do soon, after the choice about using sets is clearer!

Reply:
Tests added in 06750b7